cardinal/plugins.py

```diff
@@ -438,6 +522,8 @@ def load(self, plugins):
                 'module': module,
                 'instance': instance,
                 'commands': commands,
+                'events': events,
+                'registered_events': registered_events,
```

Comment:
Let's call this `callback_ids` instead, or I'd even be okay with `registered_callbacks` -- but I want to be careful about naming here, because an event refers to something that is emitted -- these are things that are called when an event is emitted.